sources/packages/web/src/components/aest/students/bypassedRestrictions/BypassRestrictionModal.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <v-form ref="bypassRestrictionForm">
+    <modal-dialog-base :showDialog="showDialog" title="Bypass restriction">
+      >
+      <template #content>
+        <h3 class="category-header-medium my-4">Bypass information</h3>
+        <error-summary :errors="bypassRestrictionForm.errors" />
+        <content-group>
+          <banner
+            class="mb-2"
+            :type="BannerTypes.Warning"
+            header="No active restrictions available to be bypassed or all active restrictions already have an active bypass."
+            v-if="!readOnly && restrictionsToBypass.length === 0"
+          />
+          <v-select
```

Comment:
Does the select and the radio have the `hide-details="auto"` available also?
If yes, it wold be good to keep the same spacing.